### DISCUSSION TITLE
JanusGraph release management

### DISCUSSION THREAD

**Discussion by veragini** [JanusGraph/janusgraph#3884] [Q&A]
Hi Team,

Quick question around release management: I see the latest JanusGraphDB version was release back in the beginning of 2021. How active is this project? Any future changes around how often could we expect a new release ?

Thanks,

**Comment by li-boxuan** [comment]
The latest release (1.0.0-rc2) was published on Feb 2023: https://github.com/JanusGraph/janusgraph/releases
We in general release one major version every year, together with a few minor releases.

**Reply by veragini** [reply]
Thank you for explaining it. We are in the evaluation of a few Graph database options hence why the question. Appreciate  your response.

**Comment by hadoopmarc** [comment]
Also note that JanusGraph heavily depends on Apache TinkerPop, which has frequent releases. Although JanusGraph does not follow each TinkerPop minor release, the lag was generally small in JanusGraph' s history.
